cross-project-tests/debuginfo-tests/dexter/dex/tools/test/Tool.py

```diff
@@ -217,6 +217,8 @@ def _run_test(self, test_name):
         """
         try:
             if self.context.options.binary:
```

Comment:
Is there a way to check for the darwin platform so this doesn't run on linux?
If not, this will still work fine.

Can you add a comment here explaining why this is needed (i.e., that the dsym contains the debug info)?

Reply:
Yep!